lib/loader/index.js

```diff
@@ -132,7 +132,9 @@ module.exports = {
     if (!options.forceLoad && _loadedUrls[url]) {
       // File at given url has already been loaded and option has not been set
       // to reload it, so we'll just call the callback and return.
-      callback();
+      if (typeof callback === 'function') {
```

Comment:
What happens if Javascript redefines the string constant representing the type of a function object, HUH?????